### ISSUE TITLE
[Bug]: A11Y - Dropdown selection don't have 3:1 contrast ratio with adjacent colors

### ISSUE THREAD

**Issue by nordine-marie** [microsoft/fluentui#25782]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Browser
```

### Reproduction

https://codepen.io/nordine-marie/pen/YzvLwdy

## Actual Behavior
![image](https://user-images.githubusercontent.com/76596555/203808105-b74232aa-f000-449f-85f8-e054688edafe.png)  
![image](https://user-images.githubusercontent.com/76596555/203808018-158f9068-b3a1-4616-bdf8-ade2cd4e4841.png)  
https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html  

> The visual [presentation](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#dfn-presentation) of the following have a [contrast ratio](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#dfn-contrast-ratio) of at least 3:1 against adjacent color(s)  

## Expected Behavior
Contrast between the background color of selected and other choices  should be 3:1 minimum.  

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
